pyteal/ast/subroutine_test.py

```diff
@@ -1153,6 +1153,8 @@ def mySubroutine():
             return returnValue
 
         definition = pt.SubroutineDefinition(mySubroutine, returnType)
+        evaluate_subroutine = SubroutineEval.normal_evaluator()
```

Comment:
For each of the 4 changes you made to these tests, I would like to see a version of that test with `SubroutineEval.fp_evaluator()`.

Reply:
corresponding tests added.